package.json

```diff
@@ -55,7 +55,7 @@
     "husky": "^4.3.0",
     "jest": "27.2.5",
     "jest-cli": "27.2.5",
-    "lerna": "^6.4.1",
+    "lerna": "^3.22.1",
```

Comment:
Looks like we went backward on the version here?

Reply:
nice catch!